Formula/stlink.rb

```diff
@@ -15,15 +15,13 @@ class Stlink < Formula
   end
 
   depends_on "libusb"
+  depends_on "gtk+3" => :optional
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
```

Comment:
I agree. But the real question is whether recommended must always go above optional and whether unconditional must always go above recommended :)